### ISSUE TITLE
the "jsonpatch" formatter is not compatible with the standard in its name

### ISSUE THREAD

**Issue by nlac** [benjamine/jsondiffpatch#188]
There are no "moved", "movedestination", "textDiffLine" operations in the [jsonpatch](http://jsonpatch.com) standard. See also the [rfc](https://tools.ietf.org/html/rfc6902) page.
The 3 operations above should be converted to the standard "move" and "replace" if possible.

**Comment by benjamine** [comment]
1- "the standard in its name", not really, `'jsondiffpatch'.indexOf('jsonpatch') < 0`
2- this library is prior to that standard
3- if you need it there's a formatter to convert deltas to `jsonpatch` standard (see https://github.com/benjamine/jsondiffpatch/blob/master/src/formatters/jsonpatch.js )

**Comment by nebulaszlonemethi** [comment]
Sorry i don't get the answer at all. The whole issue is about the formatter itself: https://github.com/benjamine/jsondiffpatch/blob/master/src/formatters/jsonpatch.js
So
1. `jsonpatch.js -> "jsonpatch".indexOf("jsonpatch") == 0`
2. adding the jsonpatch formatter (when that got to the repo) was prior to the standard? really?
3. thats simply not working that way, that is issue itself

Also, [this](https://github.com/benjamine/jsondiffpatch/issues/183) issue is related, i think the guy is wondering about the same.

**Comment by nlac** [comment]
reopen?

**Comment by benjamine** [comment]
sorry, my bad :man_facepalming: , I didn't notice you were reporting on the jsonpatch formatter. I see what you mean, this map seems wrong: https://github.com/benjamine/jsondiffpatch/blob/master/src/formatters/jsonpatch.js#L5 
this was an external contributor work, but I never made use of it personally,
I would welcome PRs to fix that formatter as I'm not very familiar with it, but I see it needs some help.

**Comment by aryelu** [comment]
Hi, I've added that formatter and be happy to help, can you give an example for input and output for each ? @nlac

**Comment by nlac** [comment]
Hi,
here's a simple example for both ops (i hope you asked for stg like this):
```javascript
var x = {
  a: 100,
  b: {},
  c: {
    arr: [1,2,3,4,5]
  }
}, y = {
  a: 101,
  b: {
    d: {
      arr: [1,2,3,4,5]
    }
  }
};

//standard jsonpatch diff of x -> y using the move and replace ops:
[
  {
    "op": "move",
    "from": "/c",
    "path": "/b/d"
  },
  {
    "op": "replace",
    "path": "/a",
    "value": 101
  }
]
```

**Comment by benjamine** [comment]
solved by #205
